pyproject.toml

```diff
@@ -80,6 +80,9 @@ docs = [
     "nbsphinx_link",
 ]
 
+[tool.setuptools.packages]
+find = {}
```

Comment:
I thought so, but when installing in editable mode with pixi, I needed this to avoid getting errors about all the other non-git directories I had in my pastastore folder...

Reply:
Using your explicit option it would look like this?

```TOML
[tool.setuptools.packages]
package-dir = {"" = "pastastore"}
```

Reply:
Not entirely sure. I think either that or the snippet below should do the trick:
```toml
[tool.setuptools.packages.find]
where = ["."]
include = ["pastastore"]
namespaces = false
```

Reply:
What i have now works for now, so I'll keep this in mind, but leave this PR as is.